.github/pull_request_template/Default.md

```diff
@@ -0,0 +1,6 @@
+%{all_commits}
+
+## Extra Resources
+
+- [The 10 commandments of navigating code reviews](https://techbeacon.com/app-dev-testing/10-commandments-navigating-code-reviews)
+- [Google's Code Review Developer Guide](https://google.github.io/eng-practices/review/)
```

Comment:
Can you make it so that you can also auto link the ticket that this corresponds too? if its no plausible, then its fine

Reply:
But if anything, put a template to put a link so that you could just click on it and be redirected to a ticket

Reply:
I am not able to auto-link a ticket, work project uses jira to link tickets to PRs @jmorales8

Reply:
I can add `closes [Ticket Name]` to the template and the dev can change `Ticket Name` to the one they need in the PR

Reply:
Yea thats good do that